### ISSUE TITLE
load and retrain best.pt model

### ISSUE THREAD

**Issue by ttony0321** [ultralytics/ultralytics#3144]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

i want to use the model i trained before so i load the best.pt and train about 30 epochs 
but box_loss increase 0.1182 -> 0.1923 and other loss increase too so.. did i miss something when i load the best.pt?

model = ultralytics.YOLO('yolov8s')
model = ultralytics.YOLO('yolov8m').load('results/exp5/weights/best.pt')
results = model.train(
    #model='yolov8m',
    data='/content/drive/MyDrive/mAP_data/data.yaml',
    imgsz = 960,
    epochs=30,
    batch=16,
    patience=5,
    #workers=2,
    device=0,
    project='/results',
    name='exp',
    seed=69,
    optimizer='Adam',
    val=True,
    cache=False,
    pretrained=True,
    save_period=10
)

**Comment by Kayzwer** [comment]
set warmup epochs to 0, because in warmup epochs, the lr is set to very high.

**Comment by pderrenger** [comment]
@Kayzwer you're right, increasing the warmup epochs can lead to a higher learning rate, which might explain the increase in losses you observed. Setting the warmup epochs to 0 should resolve this issue. 👍
